README.md

```diff
@@ -0,0 +1,171 @@
+# caluma-interval
+
+[![Build Status](https://travis-ci.com/projectcaluma/caluma-interval.svg?branch=master)](https://travis-ci.org/projectcaluma/caluma-interval)
+[![Coverage](https://img.shields.io/badge/coverage-100%25-brightgreen.svg)](https://github.com/projectcaluma/caluma-interval/blob/master/.coveragerc#L5)
+[![Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/projectcaluma/caluma-interval)
+
+[Caluma](https://caluma.io/) companion app for periodic usage of forms.
+
+
+## Intervalled forms
+
+There are forms you might want to use periodically. This project adds support for this
+by utilizing the `meta`-field of forms.
```

Comment:
An example would be nice.. What use case does this support?

Reply:
Absolutely! I've added one.

Reply:
Nice work on the example use case! More stories like this please :grin: